### PR TITLE
working get functions

### PR DESCRIPTION
**Pull request by rfpoulos**
@rcackerley @smith-taher  another get function, they work!  FYI, the path at the top to out local database is going to change constantly until we have a virtual machine.

### CODE REVIEW

**Comment by rfpoulos** [comment]
Also, I commented all the routes that don't have a handler.  Will uncomment them as I make handlers.

File: server.js [hunk 1]
```diff
@@ -13,13 +13,21 @@ let createUserDb = (username, password, leaning, email) => {
 };
 
 let getUserDb = (id) => {
-  return db.query(`SELECT * from users where userid = ${id}`);
+  return db.query(`SELECT * from users where articleid = ${id}`);
```
Comment (smith-taher) [inline server.js]: This return path links the article id to the user id?

Comment (rfpoulos) [reply server.js]: updated - you caught it before I pushed up the fix:-p

File: server.js [hunk 2]
```diff
@@ -48,22 +91,22 @@ let notFound = (request, response) => {
...
   { method: 'GET', path: /^\/users\/?$/, handler: getUsers },
-  { method: 'POST', path: /^\/users\/?$/, handler: postUser },
-  { method: 'POST', path: /^\/signin\/?$/, handler: signIn },
-  { method: 'DELETE', path: /^\/articles\/([0-9]+)$/, handler: deleteArticle },
+  // { method: 'POST', path: /^\/users\/?$/, handler: postUser },
+  // { method: 'POST', path: /^\/signin\/?$/, handler: signIn },
+  // { method: 'DELETE', path: /^\/articles\/([0-9]+)$/, handler: deleteArticle },
   { method: 'GET', path: /^\/articles\/([0-9]+)$/, handler: getArticle },
-  { method: 'PUT', path: /^\/articles\/([0-9]+)$/, handler: putArticle },
+  // { method: 'PUT', path: /^\/articles\/([0-9]+)$/, handler: putArticle },
```
Comment (smith-taher) [inline server.js]: Will we need a Put method for the articles as we won't be editing the articles, only displaying them via Post , Get and Delete?

**Pull request merged**